SIBR.Storage.API/Controllers/v2/UpdatesController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using NodaTime;
+using SIBR.Storage.API.Models;
+using SIBR.Storage.API.Utils;
+using SIBR.Storage.Data;
+using SIBR.Storage.Data.Models;
+using SIBR.Storage.Data.Query;
+
+namespace SIBR.Storage.API.Controllers.v2
+{
+    [ApiController]
+    [ApiVersion("2.0")]
+    [Route("v{version:apiVersion}")]
+    public class UpdatesController: ControllerBase
+    {
+        private readonly Database _db;
+        private readonly VersionStore _versionStore;
+
+        public UpdatesController(VersionStore versionStore, Database db)
+        {
+            _versionStore = versionStore;
+            _db = db;
+        }
+
+        [Route("updates")]
```

Comment:
i know we call it updates internally but this may not be the best outward-facing term, maybe "observations"?

Reply:
I was going to go with that until I saw what it was named internally. I like observations. Should I rename just the route or also the class (to `ObservationsController`)?

Reply:
i would rename everything

Reply:
it's only named updates as a holdover from discipline era v1 chron databases